lib/locator.dart

```diff
@@ -0,0 +1,15 @@
+
+import 'package:get_it/get_it.dart';
+import 'package:potato_notes/data/dao/note_helper.dart';
+import 'package:potato_notes/internal/app_info.dart';
+import 'package:potato_notes/internal/preferences.dart';
+
+import 'main.dart';
+
+final locator = GetIt.instance;
+
+void setupLocator(){
+  locator.registerSingleton<Preferences>(Preferences());
+  locator.registerLazySingleton<NoteHelper>(() => NoteHelper(db));
```

Comment:
consider using something like database.helper, you can pass db in the setupLocator params